src/community/l10n_community_roles.md

```diff
@@ -1,8 +1,16 @@
-# Roles within Mozilla l10n communities
```

Comment:
This page still needs a title (I did comment the doc that we needed one 😉 )

Reply:
I assumed that having an intro and three title sections was enough, but I can give a general title to this page if it's needed